app/views/candidates/registrations/personal_informations/_form.html.erb

```diff
@@ -37,6 +37,14 @@
                 readonly: personal_information.read_only,
                 disabled: personal_information.read_only %>
 
+          <p>
```

Comment:
Should we add this on the Manage your bookings start page `/candidates/signin` as well @MylesJarvis ?

cc @GayeCord

Reply:
I'll hold back on doing this until we decide on what to do about that page/functionality later today!